delint.sh

```diff
@@ -188,7 +188,7 @@ then
 
     # move the 'signal=' attribute back where it was
     sed -i "s/^\\(\\[connection\\)\\( from=\"[^\"]*\"\\)\\( signal=\"[^\"]*\"\\)\\(.*\\)/\\1\\3\\2\\4/g" "$IN_FILE"
-  done < <(git diff master --name-only | grep "\.tscn$")
+  done < <(git diff main --name-only | grep "\.tscn$")
```

Comment:
Mention this change in the commit comment

Reply:
Expanded commit comment, `Updated scripts which target 'master' branch to target 'main' instead.`